alg/sha256_arm.c

```diff
@@ -108,41 +101,30 @@ SHA256_Transform_arm(uint32_t state[static restrict 8],
 	Y[3] = vld1q_u32(&W[12]);
 
 	/* 2. Initialize working variables. */
-	memcpy(S, state, 32);
+	S0 = vld1q_u32(&state[0]);
```

Comment:
I would write this

```
S[0] = _state[0] = vld1q_u32(&state[0]);
S[1] = _state[1] = vld1q_u32(&state[4]);
```

unless there's a good reason not to.

Reply:
done